Samples/Server/Directx-Multithreaded/MultithreadedTestRunner.vcxproj

```diff
@@ -116,7 +116,7 @@
       <ExceptionHandling>Sync</ExceptionHandling>
       <AdditionalIncludeDirectories>$(ProjectDir);..\..\..\Libraries\DXUT\Core;..\..\..\Libraries\DXUT\Optional;..\..\..\Libraries\DXUT\Remoting;..\..\..\Utilities\VideoTestRunner\inc;..\..\..\Libraries\NvEncoder\inc;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
       <AdditionalOptions> %(AdditionalOptions)</AdditionalOptions>
-      <PreprocessorDefinitions>SERVER_APP;TEST_RUNNER;_CRT_SECURE_NO_WARNINGS;_DEBUG;_WINDOWS;WIN32;_WIN32_WINNT=0x0602;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>WEBRTC_WIN;WIN32_LEAN_AND_MEAN;SERVER_APP;TEST_RUNNER;_CRT_SECURE_NO_WARNINGS;_DEBUG;_WINDOWS;WIN32;_WIN32_WINNT=0x0602;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
I think that we can safely remove WEBRTC_WIN;WIN32_LEAN_AND_MEAN;SERVER_APP flags now.

Reply:
oh oops - thought i did that. will do right now